src/Adapter/MSTest.TestAdapter/Discovery/UnitTestDiscoverer.cs

```diff
@@ -1,6 +1,7 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This helps me measure, I will revert, or we can log it?

Reply:
PR is ready for review so this part should be fixed.